packages/wp-now/src/run-cli.ts

```diff
@@ -0,0 +1,78 @@
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { startServer } from './start-server';
+import { portFinder } from './port-finder';
+import { DEFAULT_PHP_VERSION } from './constants';
+import { SupportedPHPVersion } from '@php-wasm/universal';
+
+function startSpinner(message: string) {
+	process.stdout.write(`${message}...\n`);
+	return {
+		succeed: (text: string) => {
+			console.log(`${text}`);
+		},
+		fail: (text: string) => {
+			console.error(`${text}`);
+		},
+	};
+}
+
+export async function runCli() {
+	const port = await portFinder.getOpenPort();
+	return yargs(hideBin(process.argv))
+		.scriptName('wp-now')
+		.usage('$0 <cmd> [args]')
+		.command(
+			'start',
+			'Start the server',
+			(yargs) => {
+				yargs.option('port', {
+					describe: 'Server port',
+					type: 'number',
+					default: port,
+				});
+				yargs.option('path', {
+					describe:
+						'Path to the PHP or WordPress project. Defaults to the current working directory.',
+					type: 'string',
+					default: process.cwd(),
+				});
+				yargs.option('wp-content', {
```

Comment:
I think this argument is confusing. Can we remove it for now, and re-think it at a later point?

Reply:
Removed! https://github.com/WordPress/wordpress-playground/pull/188/commits/55189e4b160c9835a1f13371389a79eb0803c550

The goal of the parameter was to map the wp-content folder to a diferent path.
Currently we are mapping it to: `~/.wp-now/wp-content/${projectName}-${sha1(projectPath)}`